bottom-navigator/src/main/java/com/pandora/bottomnavigator/BottomNavigator.kt

```diff
@@ -47,6 +47,11 @@ open class BottomNavigator internal constructor() : ViewModel() {
      */
     open fun resetRootFragmentCommand(): Observable<Fragment> = resetRootFragmentSubject.hide()!!
 
+    /**
+     *  Returns if resetRootFragmentSubject has any observer
+     */
+    open fun hasObserversResetRootFragmentCommand() = resetRootFragmentSubject.hasObservers()
```

Comment:
this name is kind of awkward. I think it's better the other way around `resetRootFragmentCommandHasObservers`